src/main/java/ladder/structure/Lines.java

```diff
@@ -0,0 +1,49 @@
+package ladder.structure;
+
+import ladder.game.Points;
+import ladder.structure.connection.ConnectionStrategy;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Lines {
+    private static final String LADDER_HEIGHT_EXCEPTION = "사다리게임의 높이는 0 이상입니다.";
+    private List<LineOfLadder> lines;
+
+    public Lines(int personCount, int ladderHeight, ConnectionStrategy connectionStrategy) {
+        verityLadderHeight(ladderHeight);
+        this.lines = new ArrayList<>();
+        addLines(personCount, ladderHeight, connectionStrategy);
+    }
+
+    private void addLines(int personCount, int ladderHeight, ConnectionStrategy connectionStrategy) {
+        Points points = new Points(personCount - 1);
```

Comment:
"참여할 사람의 수만큼 `Point`를 생성한다."라는 요구 사항에는 아래의 코드가 더 적절해 보여요.

```suggestion
        Points points = new Points(personCount);
```